docs/guide/troubleshooting.md

```diff
@@ -83,6 +101,10 @@ If HMR is not handled by Vite or a plugin, a full reload will happen.
 
 Also if there is a dependency loop, a full reload will happen. To solve this, try removing the loop.
 
+### High number of HMR updates in console
+
+This can be caused by a dependency loop,. To solve this, try removing the loop.
```

Comment:
line 102 indicates something else will happen when there's a circular dependency
this is normally referred to as a "circular dependency" rather than "dependency loop". if we keep both lines we should probably use that wording. also there was a `,.` here which should be updated
```suggestion
This can be caused by a circular dependency. To solve this, try breaking the loop.
```